apsuite/commissioning_scripts/loco.py

```diff
@@ -1241,6 +1311,23 @@ def _handle_dip_fit_ks(self, fname_jloco_ks_dip):
                 self._jloco_ks_dip = self._convert_dict2array(
                     jloco_ks_dip_dict, 'dipole', is_normal=False)
 
+    def _handle_skewquad_fit_ks(self, fname_jloco_ks_skewquad):
+        # calculate Ks jacobian for skew quadrupoles
+        if self.config.fit_skew_quadrupoles:
+            if fname_jloco_ks_skewquad is None:
+                print('calculating skew quadrupoles ksmatrix...')
+                t0 = time.time()
```

Comment:
pylint 3-letter minimum name rule. suggestions: `time0`, `dtime`

Reply:
ok, I followed all pylint suggestions in the last commit, except one: `pylint(too-many-lines)`. But I hope to solve that problem soon, breaking the loco.py in smaller subfiles.

Reply:
Yes, `too-many-lines`, `too-many-arguments` and `too complex` are harder to follow. we comply gradually. (in general I agree with these too! every time I took the time to fix `too complex` I ended up with the filling that code was easier to read)

Reply:
I'll keep this unresolved so you have a better change to read my comment. but you can resolve it and merge after you read this.